File: source/includes/connect/crl-file.php
$uri = 'mongodb://<hostname>:<port>';

$uriOptions = [
   'tls' => true,
];

$driverOptions = [
    'crl_file' => '/path/to/file.pem'
];

$client = new MongoDB\Client($uri, $uriOptions, $driverOptions);

File: content/cpp-driver/current/source/includes/connect/crl-file.cpp
mongocxx::options::client client_options;
mongocxx::options::tls tls_options;

tls_options.crl_file("/path/to/file.pem");
client_options.tls_opts(tls_options);

mongocxx::uri uri("mongodb://<hostname>:<port>/?tls=true");
mongocxx::client client(uri, client_options);